pages/game.js

```diff
@@ -19,7 +24,25 @@ export default class Index extends Component {
         isVictory: false,
     };
 
-    handleClick = () => {};
+    handleClick = tile => {
+        const { currentGrid, resolvedGrid, turn } = this.state;
+
+        try {
+            const coordsTileToMove = findTileByValue(currentGrid, tile);
+            const newCurrentGrid = move(currentGrid, coordsTileToMove);
+            const isVictory = areGridsEquals(newCurrentGrid, resolvedGrid);
+
+            const newState = {
+                currentGrid: newCurrentGrid,
+                isVictory,
+                turn: turn + 1,
+            };
+
+            this.setState(newState);
+        } catch (error) {
+            console.error(error);
```

Comment:
A silenced error is usually a code smell. Either let it throw or add a comment justifying why it's there.

Reply:
The two methods `findTileByValue` and `move` could return an error. For example, it's the case if the user try to move a wrong tile.

I don't have a fancy way to display this error at the moment, so I prefer to do nothing. Do you think it's a priority ?

Reply:
Indeed, I don't think it is a priority, you can add a `//TODO` comment explaining why.